tests/test_flake8_trio.py

```diff
@@ -90,9 +93,7 @@ def test_eval(test: str, path: str):
 
         # add command-line args if specified with #ARGS
         elif reg_match := re.search(r"(?<=ARGS).*", line):
-            for arg in reg_match.group().split(" "):
-                if arg.strip():
-                    parsed_args.append(arg.strip())
+            parsed_args.append(reg_match.group().strip())
```

Comment:
Changed this to handle spaces in a command line arg - this means that the name `ARGS` is misleading but that is renamed in #91 (rip me bundling changes again).
This leads to some minor merge conflicts, but I'll likely rebase #91 on top of this one to fix those.

Reply:
Sounds good - I'll next have internet around Jan 9th, so don't panic if review is a bit delayed.

Reply:
gtk, I'll rebase and fix merge conflicts in it. May also split it up if I start messing with stuff in other PR's that might lead to conflicts.